rotkehlchen/tests/api/test_assets.py

```diff
@@ -857,3 +857,51 @@ def test_search_assets_with_levenshtein(rotkehlchen_api_server):
         },
     )
     assert_error_response(response, contained_in_msg='Failed to deserialize ChainID value near')
+
+
+def test_asset_information_by_identifier(rotkehlchen_api_server):
+    # test asking for a single evm token
+    response = requests.post(
+        api_url_for(
+            rotkehlchen_api_server,
+            'assetsresource',
+        ),
+        json={
+            'identifiers': [A_DAI.identifier],
+        },
+    )
+    result = assert_proper_response_with_result(response)
+    assert result[0]['identifier'] == A_DAI
+    assert 'address' in result[0]
+    assert 'decimals' in result[0]
+    assert result[0]['type'] == AssetType.EVM_TOKEN.serialize()
+
+    # ask for a crypto asset and a fiat asset
+    response = requests.post(
+        api_url_for(
+            rotkehlchen_api_server,
+            'assetsresource',
+        ),
+        json={
+            'identifiers': [A_BTC.identifier, A_USD.identifier],
```

Comment:
Also I would add a check for a custom asset query

Reply:
done